lib/main.dart

```diff
@@ -3,107 +3,113 @@ import 'package:flutter/material.dart';
 void main() => runApp(new MyApp());
 
 class MyApp extends StatelessWidget {
-  // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
     return new MaterialApp(
-      title: 'Flutter Demo',
```

Comment:
We have removed title. Should it be kept with appropriate value for completeness?

Reply:
It doesn't matter. our app does not have the title anyway.

Reply:
Fine.